src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -821,6 +821,9 @@ Do you want to proceed?</value>
   <data name="Applying_changes" xml:space="preserve">
     <value>Applying changes</value>
   </data>
+  <data name="String_Escape_Character" xml:space="preserve">
+    <value>String - Escape Character</value>
```

Comment:
This could "String - Escape Character" or "String - Escape Sequence".  This is a user visible string, so i don't mind some bike-shedding.

Reply:
@CyrusNajmabadi  Is this representing the `\` part, or the full sequence `\u0001`. If it the former then use character, otherwise sequence.